client/signup/steps/domains/index.jsx

```diff
@@ -420,12 +419,8 @@ class DomainsStep extends React.Component {
 			: translate( "Enter your site's name or some keywords that describe it to get started." );
 	}
 
-	isDomainsFirstFlow() {
```

Comment:
We can safely drop this, as it's actually what `isDomainOnly` prop represents. :)